### PR TITLE
PD-554 DR docs fixes

### PR DESCRIPTION
**Pull request by nathan-px**

### CODE REVIEW

**Comment by nathan-px** [comment]
@Rohit-PX See comments on the google sheet

File: content/portworx-install-with-kubernetes/migration/_index.md [hunk 1]
```diff
@@ -1,10 +1,317 @@
 ---
-title: "Migration"
+title: "Migration with Stork on Kubernetes"
 linkTitle: "Migration"
-keywords: cloud, backup, restore, snapshot, DR, migration, kubemotion
+keywords: cloud, backup, restore, snapshot, DR, migration, migration
```
Comment (Rohit-PX) [inline content/portworx-install-with-kubernetes/migration/_index.md:4]: we have migration twice.

File: content/portworx-install-with-kubernetes/disaster-recovery/async-dr/_index.md [hunk 2]
```diff
@@ -42,6 +42,59 @@ The following Kubernetes resources are supported as part of the Asynchronous DR
...
+
+## Add CRDs
+
+If Portworx doesn't have the CRD you need, you can register a new one. To do this, create an `ApplicationRegistration` spec, specifying your own values for the following:
+
+* **metadata.name:** 
+* **resources.PodsPath:** With the pod path in case pods has to be deleted for disabling CR <!-- what does this mean? -->
+* **resources.group:** With the CRD the group goes into
+* **resources.version:** With version of this CRD
+* **resources.kind** <!-- what does this mean? -->
```
Comment (nathan-px) [inline content/portworx-install-with-kubernetes/disaster-recovery/async-dr/_index.md:73]: @Rohit-PX need the descriptions for these

File: content/portworx-install-with-kubernetes/disaster-recovery/async-dr/_index.md [hunk 3]
```diff
@@ -42,6 +42,59 @@ The following Kubernetes resources are supported as part of the Asynchronous DR
...
+* **resources.keepStatus:** <!-- what does this mean? -->
+* **resources.suspendOptions.path:** <!-- what does this mean? -->
+* **resources.suspendOptions.type:** <!-- what does this mean? -->
+
+```text
+apiVersion: stork.libopenstorage.org/v1alpha1
+kind: ApplicationRegistration
+metadata:
+  name: myappname
+# list of CR for <myappname> <!-- what does this mean? does this pertain to `name: myappname`? or is it denoting other potential metadata fields? -->
```
Comment (nathan-px) [inline content/portworx-install-with-kubernetes/disaster-recovery/async-dr/_index.md:83]: @Rohit-PX I also have questions about the #comments here, see `<!--` inline comments below.

**Pull request merged**